File: fizz_buzz.c
#include <stdio.h>
//#include <stdlib.h>

int main(void)
{
	int i;
  
	for(i=1; i<101; i++)
	{
		switch(i % 15)
		{
			case 0:
				printf("fizzbuzz\n");
				break;

			case 5:
			case 10:
				printf("buzz\n");
				break;

			case 3:
			case 6:
			case 9:
			case 12:
				printf("fizz\n");
				break;

			default:
				printf("%d\n", i);
		}
	}
	return 0;
}

File: fizz_buzz.py
def fizzbuzz(i):
    if(i % 15 == 0):
        return "fizzbuzz"
    if(i % 3 == 0):
        return "fizz"
    if(i % 5 == 0):
        return "buzz"
    return str(i)

result = [fizzbuzz(i) for i in range(1,101)]
print("\n".join(result))
